### PR TITLE
Add command and command autocompletion support for TextChatService

### PR DESCRIPTION
**Pull request by ccuser44**

### CODE REVIEW

**Comment by WalkerOfBacon** [comment]
I will test this

File: MainModule/Server/Core/Admin.lua [hunk 1]
```diff
@@ -1244,6 +1244,58 @@ return function(Vargs, GetEnv)
...
+						if v:IsA("TextChatCommand") then
+							blacklistedCommands[v.PrimaryAlias] = true
+							blacklistedCommands[v.SecondaryAlias] = true
+						end
+					end
+
+					for name, data in Commands do
+						local command1, command2 = nil, nil
+
+						for _, v in data.Commands do
```
Comment (WalkerOfBacon) [inline MainModule/Server/Core/Admin.lua:1271]: image

**Review by WalkerOfBacon — CHANGES_REQUESTED** [review]

File: MainModule/Server/Core/Admin.lua (hunk 1 above)
Comment (ccuser44) [reply MainModule/Server/Core/Admin.lua:1271]: I cannot reproduce this issue. Are you using any plugins?

**Comment by Dimenpsyonal** [comment]
Wait... Why does it need the regular prefix as well
Screenshot 2023-10-29 at 11 05 16

**Comment by watameln** [comment]
Screenshot 2023-10-29 at 11 05 16](https://github.com/Epix-Incorporated/Adonis/assets/64012878/64d2c3be-34e4-4c59-9649-15d80ee75039%22%3E)

**Comment by Dimenpsyonal** [comment]
what

File: MainModule/Server/Core/Admin.lua (hunk 1 above)
Comment (WalkerOfBacon) [reply MainModule/Server/Core/Admin.lua:1275]: I just got back to this-

Anyways no I wasn't

**Comment by ccuser44** [comment]
> Wait... Why does it need the regular prefix as well
> ￼

Roblox limitation

**Pull request merged**